spec/feature/login_spec.rb

```diff
@@ -190,27 +190,67 @@
       )
     end
 
-    before do
-      expect(user.ephemeral_states.last).to be_nil
+    context "when the feature_flag_enforce_levels_of_authentication is not 'enabled'" do
+      before do
+        expect(user.ephemeral_states.last).to be_nil
         log_in
-    end
+      end
 
-    it "records that the user has logged in with level-of-authentication 1" do
-      enter_mfa
+      it "records that the user has logged in with level-of-authentication 1" do
+        enter_mfa
+
+        expect(user.reload.ephemeral_states.last&.level_of_authentication).to eq("level1")
+      end
+
+      context "when the user doesn't have MFA set up" do
+        let(:user) { FactoryBot.create(:user, :without_mfa) }
 
-      expect(user.reload.ephemeral_states.last&.level_of_authentication).to eq("level1")
+        it "records that the user has logged in with level-of-authentication 0" do
+          expect(user.reload.ephemeral_states.last&.level_of_authentication).to eq("level0")
+        end
+      end
     end
 
-    context "when the user doesn't have MFA set up" do
-      let(:user) { FactoryBot.create(:user, :without_mfa) }
+    context "when the feature_flag_enforce_levels_of_authentication is 'enabled'" do
+      context "for a level0 authorised user" do
+        before do
+          allow(Rails.configuration).to receive(:feature_flag_enforce_levels_of_authentication).and_return(true)
+          application.update!(scopes: "openid level0")
```

Comment:
Rather than doing `application.update!` here (and below), make its scopes `openid level0 level1` in the `let`